content/docs/user-guide/interactivity/robotics/joints-manipulation.md

```diff
@@ -0,0 +1,107 @@
+- Control systems based on articulations as well as hinge and prismatic joints.
+- Extend interfaces for developers to implement your own control components through __JointsManipulationRequests__ and __JointsPositionControllerRequests__.
+- Use manipulation components in multi-robot scenarios.
+
+### Limitations
+
+Manipulation components only work with single degree of freedom joints.
+
+### Components and their interactions
+
+Movement of joints is handled through a set of interfaces and components that implement them.
+
+| Interface                            | Components                                                                     | Role                                                                                  |
+|--------------------------------------|--------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| __JointsPositionControllerRequests__ | __JointsArticulationControllerComponent__<br/>__JointsPIDControllerComponent__ | Move joints towards desired positions.                                                |
+| __JointsManipulationRequests__       | __JointsManipulationComponent__                                                | Hold and publish joints state information, relay commands to controllers              |
+| __JointsTrajectoryRequests__         | __JointsTrajectoryComponent__                                                  | Host action server for trajectory commands, control trajectory through goal positions |
+
+## Simulating joint systems
+
+### Start quickly with Manipulation Template
+
+If your goal is to simulate a robotic arm, the quickest way to start is to [create a project](/docs/welcome-guide/create/) from Manipulation Template.
+
+If you followed the steps of [project configuration](project-configuration.md), your environment variables for `O3DE_HOME` and `PROJECT_PATH` will be set, and the templates registered.
+Create a new project with the following command:
+
+```shell
+${O3DE_HOME}/scripts/o3de.sh create-project --project-path $PROJECT_PATH --template-name Ros2RoboticManipulationTemplate
+```
+
+The template comes with a couple of examples which you can run following its [README](https://github.com/o3de/o3de-extras/tree/development/Templates/Ros2RoboticManipulationTemplate).
+
+### Configure your robot
+
+#### Import the robot
+Once your project is up, use the [robot import](importing-robot.md) feature to load your robot of choice into O3DE.
+Make sure that the checkbox ```Use articulation for joints and rigid bodies``` is checked on the last page of the importer.
+Using articulations is strongly recommended for stable simulation of robotic arms and other joint systems.
+
+#### Add necessary components
+If the import proceeded without issues, open the root entity of your newly created prefab and add three new components:
```

Comment:
"If the import proceeded without issues' - can we refer to some other documentation otherwise?

Reply:
Currently we do not have such a documentation (everything is in Robot Importer as linked), but I have requested troubleshooting documentation as a part of new changes to the importer.